### PR TITLE
Configure travis

### PR DESCRIPTION
**Pull request by nea1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# flyte client
+# flyte-client
+
+![Build Status](https://travis-ci.org/HotelsDotCom/flyte-bamboo.svg?branch=master)
```
Comment (ukjasonwright) [inline README.md]: flyte-bamboo?

Comment (ukjasonwright) [inline README.md]: everyone else has this above the title I think?

Comment (nea1) [reply README.md]: https://github.com/kubernetes/kubernetes ;)

Comment (nea1) [reply README.md]: updated

**Review by ukjasonwright — APPROVED** [review]

**Review by pamelin — APPROVED** [review]

**Pull request merged**